hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/VertexAPI.java

```diff
@@ -259,22 +313,53 @@ private static void checkBatchSize(HugeConfig config,
         int max = config.get(ServerOptions.MAX_VERTICES_PER_BATCH);
         if (vertices.size() > max) {
             throw new IllegalArgumentException(String.format(
-                      "Too many counts of vertices for one time post, " +
+                      "Too many vertices for one time post, " +
                       "the maximum number is '%s'", max));
         }
     }
 
-    @JsonIgnoreProperties(value = {"type"})
-    private static class JsonVertex implements Checkable {
+    private Id getVertexId(HugeGraph g, String labelId, JsonVertex vertex) {
+        assert g.vertexLabel(labelId).idStrategy() != IdStrategy.AUTOMATIC;
+
+        List<Id> pkIds = g.vertexLabel(vertex.label).primaryKeys();
```

Comment:
prefer using if-else according to the id strategy, like
```
if (CUSTOMIZED) {
    xxx
} else {
    assert PRIMARY_KEY
    xxx
}
```
Clear logic is better than code clean